tldw_Server_API/app/core/AuthNZ/repos/mfa_repo.py

```
@@ -0,0 +1,294 @@
+                        """,
+                        (
+                            encrypted_secret,
+                            backup_codes_json,
+                            updated_at.isoformat(),
+                            user_id,
+                        ),
+                    )
+                    try:
+                        await conn.commit()
+                    except Exception:
+                        pass
```

Ruff (0.14.7)

`try`-`except`-`pass` detected, consider logging the exception

(S110)